k8s/namespaces/pcq/pcq-backend/demo.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pcq-backend
+spec:
```

Comment:
You dont need to add thes files if they are not needed.

Reply:
I thought as much, ok I'll remove. One of them has an environment value which we'll need to keep. Thanks.